test/acceptance/features/steps/steps.py

```diff
@@ -200,6 +200,10 @@ def sbr_is_applied(context):
         ns = None
     assert sbr.create(sbr_yaml, ns) is not None, "Service binding not created"
 
+    time.sleep(10)
```

Comment:
we should not wait for absolute time.. because it could be underestimated, it depends really on environment.

Reply:
Fixed

Reply:
1f140ad

Reply:
Done here b7139c7